docs/graphql/manual/index.rst

```diff
@@ -12,12 +12,62 @@ Hasura GraphQL engine documentation
   :depth: 1
   :local:
 
-The Hasura GraphQL engine lets you set up a GraphQL server and event triggers over a Postgres database in minutes.
+The Hasura GraphQL engine connects to your databases & microservices and auto-generates a production-ready GraphQL backend in minutes.
 
+.. list-table::
+   :header-rows: 1
+   :class: toc-table
+
+   * - Basics
+     - Business Logic
+     - Auth
+   * - `Getting Started <getting-started/index>`_
+     - `Actions <actions/index>`_
+     - `Authentication <auth/authentication/index>`_
+   * - `Schema <schema/index>`_
+     - `Remote Schemas <remote-schemas/index>`_
+     - `Authorization <auth/authorization/index>`_
+   * - `Queries <queries/index>`_
+     - `Event Triggers <event-triggers/index>`_
+     - `Access Control Basics <auth/authorization/basics>`_
```

Comment:
@sezgi it might be a bit confusing to have `Authentication` and `Authorization` and then the other 3 on the same level even though they belong under `Authorization`. From the structure it looks like they are all on the same hierarchy. Maybe `Authentication` and `Authorization` is enough?

Reply:
I thought of these more as shortcuts than as a hierarchical nav. Maybe we can refer to traffic data for this to see the most used ones (I'll check today).

Reply:
Also only two links would make the list too short visually

Reply:
changed the auth section to highlight authn sections instead of authz sections (based on traffic data)